7_zebra/zebra_scip.py

```diff
@@ -0,0 +1,102 @@
+"""
+Solution to the Zebra
+
+This version uses SCIP as a solver.
+
+Created by Éder Pinheiro and Eric Zettermann  (Sep 9, 2021), MipMaster.org.
+"""
```

Comment:
This is now the SCIP version of `zebra_pulp_aster`

Reply:
Nice.